### PR TITLE
Maven task

### PR DESCRIPTION
**Pull request by ViktoriiaLozova**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <artifactId>project</artifactId>
+    <packaging>jar</packaging>
+    <version>1.3</version>
+    <name>project</name>
+    <url>http://maven.apache.org</url>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>3.1.0</version>
```
Comment (AntonApanasovych) [inline pom.xml]: [Good Practice]: keep versioning in properties section, easier to maintain

Example:
```
 <properties>
        <rest-assured.version>3.0.4</rest-assured.version>
        <junit.version>4.12</junit.version>
        <testng.version>6.11</testng.version>
        <nimbus-jose-jwt.version>4.27</nimbus-jose-jwt.version>
        <slf4.version>1.7.21</slf4.version>
        <swagger-request-validator.version>1.2.1</swagger-request-validator.version>
        <aspectj.version>1.8.10</aspectj.version>
        <json-unit.version>1.28.2</json-unit.version>
        <selenide.version>4.8</selenide.version>
        <guava.version>23.0</guava.version>
        <jsoup.version>1.8.1</jsoup.version>
        <xsoup.version>0.3.1</xsoup.version>
        <hibernate-core.version>5.2.11.Final</hibernate-core.version>
        <powermock.version>1.7.1</powermock.version>
    </properties>
```

File: src/main/java/org/apache/maven/archetypes/App.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package org.apache.maven.archetypes;
+
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+
+public class App 
+{
```
Comment (AntonApanasovych) [inline src/main/java/org/apache/maven/archetypes/App.java]: Bad formatting. Open block sign `{` on new line is c# style. In Java code convention should be on same raw as declaration.
Please look in your IDE settings Keymap and find `Reformat Code` hot key. Apply it every time before commit.

File: src/main/java/org/apache/maven/archetypes/App.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    final static Logger logger = LogManager.getLogger(App.class);
+	
+    public static void main( String[] args )
+    {
```
Comment (AntonApanasovych) [inline src/main/java/org/apache/maven/archetypes/App.java]: bad formatting

File: src/test/java/org/apache/maven/archetypes/AppTest.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
+    extends TestCase
+{
```
Comment (AntonApanasovych) [inline src/test/java/org/apache/maven/archetypes/AppTest.java]: bad formatting

File: pom.xml (hunk 1 above)
Comment (ViktoriiaLozova) [reply pom.xml]: done

File: src/main/java/org/apache/maven/archetypes/App.java (hunk 2 above)
Comment (ViktoriiaLozova) [reply src/main/java/org/apache/maven/archetypes/App.java]: done

File: src/main/java/org/apache/maven/archetypes/App.java (hunk 3 above)
Comment (ViktoriiaLozova) [reply src/main/java/org/apache/maven/archetypes/App.java]: done

File: src/test/java/org/apache/maven/archetypes/AppTest.java (hunk 4 above)
Comment (ViktoriiaLozova) [reply src/test/java/org/apache/maven/archetypes/AppTest.java]: done

**Pull request merged**